timApp/static/templates/manage.html

```diff
@@ -72,13 +90,19 @@
     </div>
 </div>
 
-<bootstrap-panel title="Relevance" ng-if="$ctrl.item.rights.editable">
+<bootstrap-panel title="Relevance" ng-if="$ctrl.item.rights.editable" showHeadingAnchors="true" anchorID="relevance">
```

Comment:
Kun Angular-komponentteja käytetään AngularJS:ssä, attribuutit pitää kirjoittaa kebab-casessa (katso [Using Angular Components from AngularJS Code
](https://angular.io/guide/upgrade#using-angular-components-from-angularjs-code), kohta "Use kebab-case for downgraded component attributes".

Seuraten muunnossääntöjä, pitää siis olla

```suggestion
<bootstrap-panel title="Relevance" ng-if="$ctrl.item.rights.editable" [show-heading-anchors]="true" anchor-i-d="relevance">
```

Tuossa `anchorID` muuntuu muotoon `anchor-i-d`, koska sekä I että D ovat isolla. Se voisi toki samalla muuntaa muotoon `anchorId` niin voisi kirjoittaa oikeellisemmin `anchor-id`.

Reply:
Joo nyt kun mainitsit, niin olinhan minä joskus tuonkin dokumentaation lukenut, muttei tehdessä vaan muistunut mieleen :)

Reply:
Nyt on korjattu.